refinery/tool_manager/views.py

```diff
@@ -72,22 +71,18 @@ def get_queryset(self):
         authenticated user has read permissions on.
         """
         user_tools = []
-        try:
-            user_tools.extend(
-                get_objects_for_user(
-                    self.request.user,
-                    "tool_manager.read_workflowtool"
-                )
+        user_tools.extend(
+            get_objects_for_user(
+                self.request.user,
+                "tool_manager.read_workflowtool"
             )
-            user_tools.extend(
-                get_objects_for_user(
-                    self.request.user,
-                    "tool_manager.read_visualizationtool"
-                )
+        )
+        user_tools.extend(
+            get_objects_for_user(
+                self.request.user,
+                "tool_manager.read_visualizationtool"
             )
-        except GuardianError as e:
-            return HttpResponseBadRequest(e)
-
+        )
```

Comment:
What changed? Does something else catch the GuardianError, or was that a mistake before?

Reply:
This was a mistake before